src/Utopia/Messaging/Adapter/Push.php

```diff
@@ -17,6 +17,11 @@ public function getMessageType(): string
         return PushMessage::class;
     }
 
+    protected static function getExpiredErrorMessage(): string
+    {
+        return 'Expired device token.';
+    }
```

Comment:
Why use a static function instead of a constant?

Reply:
Following the same convention. 
It's same as getType method which returns a const string.

Reply:
@stnguyen90 message type and this can both return constants.